src/interface/widget.c

```diff
@@ -91,7 +93,16 @@ void widget_scroll_update_thumbs(rct_window *w, int widget_index)
 		y += 11;
 		view_size += 10;
 		scroll->v_thumb_bottom = min(y, view_size);
+
+		if(scroll->v_thumb_bottom - scroll->v_thumb_top < 20) {
+			double percent = (scroll->v_thumb_bottom*1.0)/view_size;
+			printf("percent: %f\n", percent);
```

Comment:
Yes, but this is only a draft. If this is approved, then I'll do the same for the horizontal scrollbar and some code cleaning